src/App/Hovedside/Sak/useSaker.tsx

```diff
@@ -32,6 +51,24 @@ const HENT_SAKER: TypedDocumentNode<SakerResultat> = gql`
                     paaminnelseTidspunkt
```

Comment:
brukes fortsatt disse? `oppgaver` og `frister`? Eller har `tidslinje` tatt over for dem? I så fall kan vi slette herfra, så kan vi også bli kvitt dette fra bruker-api.